src/main/java/hudson/plugins/ec2/win/EC2WindowsLauncher.java

```diff
@@ -91,10 +91,11 @@ protected void launchScript(EC2Computer computer, TaskListener listener) throws
 
             logger.println("remoting.jar sent remotely. Bootstrapping it");
 
+            final String javaPath = node.javaPath;
             final String jvmopts = node.jvmopts;
             final String remoteFS = WindowsUtil.quoteArgument(node.getRemoteFS());
             final String workDir = Util.fixEmptyAndTrim(remoteFS) != null ? remoteFS : tmpDir;
-            final String launchString = "java " + (jvmopts != null ? jvmopts : "") + " -jar " + tmpDir + AGENT_JAR + " -workDir " + workDir;
+            final String launchString = javaPath + " " + (jvmopts != null ? jvmopts : "") + " -jar " + tmpDir + AGENT_JAR + " -workDir " + workDir;
```

Comment:
Note: unlike the Mac & Unix versions, this launcher doesn't allow specifying command `prefix` or `suffix`